### PR TITLE
Publishing to npm

### PR DESCRIPTION
**Pull request by edmoss345**
Small changes made to publish to npm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
     "start": "node add_safeguarding_to_flows.js"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/fagiothree/safeguarding-rapidpro.git"
+    "url": "git+https://github.com/IDEMSInternational/safeguarding-rapidpro.git"
   },
-  "author": "",
+  "author": "IDEMS",
   "license": "ISC",
```
Comment (istride) [inline package.json]: GPL-3.0-only

File: index.js
Comment (istride) [inline index.js:1]: I think it's a bit odd to do leave this file empty. This should be the entry point into this library, for example, what do want to happen when someone writes:
```
const x = require('safeguarding-rapidpro');
```

I can appreciate that the code in this repo needs refactoring before we can do anything meaningful in this file (exporting functions), but I hope there is an intention to do so in the future.

**Review by istride — CHANGES_REQUESTED** [review]

File: index.js
Comment (edmoss345) [reply index.js:1]: Yes agreed, mainly just put this in as a placeholder so we are pointing to a clear entry point in the 'package.json' file, have added a comment just explaining that this file should be improved in future development. Ideal future state is that the functions we want to expose are referenced and can be called as a library or package

File: package.json (hunk 1 above)
Comment (edmoss345) [reply package.json]: changed

**Pull request merged**